### ISSUE TITLE
`#region` code folding doesn't work if there's a multi-line comment inside it

### ISSUE THREAD

**Issue by disarray2077** [beefytech/Beef#1483]
Example code:
```cs
#region Usings
/* Usings
*/
using System;
#endregion
```

**Comment by disarray2077** [comment]
Looks like this issue is solved, but I found two more issues related to the code folding so I will report them here:

- `#if false` is only foldable if you include a comment inside it
- `#if true` is foldable if you include one comment in it and another comment inside it, but it will fold wrongly

Code:
```cs
#if false
			//ghgjg
			Console.WriteLine("hi");
#endif

#if true	//sadad
			//ghgjg
			Console.WriteLine("hi");
#endif
```

**Comment by bfiete** [comment]
Fixed at 9b7fc5564efc947a33c7bc88447fa1cd384f16ed
